examples/02-plot/spherical.py

```diff
@@ -34,7 +35,8 @@ def _cell_bounds(points, bound_position=0.5):
     >>> cell_bounds(a)
     array([-1.25, -0.75, -0.25,  0.25,  0.75,  1.25,  1.75,  2.25])
     """
-    assert points.ndim == 1, "Only 1D points are allowed"
+    if points.ndim != 1:
+        raise ValueError("Only 1D points are allowed.")
```

Comment:
I just found google style guide talking about how [you shouldn't use exclamation points](https://developers.google.com/style/exclamation-points).  Thanks for changing that.

Reply:
My pleasure, but I don't think I changed any of that :smile: